internal/client/connutils.go

```diff
@@ -34,10 +38,19 @@ import (
 
 	"github.com/edgedb/edgedb-go/internal/edgedbtypes"
 	"github.com/edgedb/edgedb-go/internal/snc"
+	"github.com/sigurn/crc16"
 )
 
-var isDSNLike = regexp.MustCompile(`(?i)^[a-z]+://`)
-var isIdentifier = regexp.MustCompile(`^[A-Za-z_][A-Za-z_0-9]*$`)
+var (
+	isDSNLike      = regexp.MustCompile(`(?i)^[a-z]+://`)
+	instanceNameRe = regexp.MustCompile(
+		`^([A-Za-z_]\w*)(?:/([A-Za-z_]\w*))?$`,
+	)
+	crcTable       *crc16.Table = crc16.MakeTable(crc16.CRC16_XMODEM)
+	base64Encoding              = base64.StdEncoding.WithPadding(
```

Comment:
```suggestion
	base64Encoding              = base64. URLEncoding.WithPadding(
```

I think this is the alternative used in JWT